CHECKENFORCER

```diff
@@ -0,0 +1,3 @@
+format: v0.1-alpha
+enabled: true
```

Comment:
Shouldn't the presence of this config file indicate that it is enabled? It seems like we could remove some redundancy and make enabled=true by default and still allow folks to disable it by setting it to false.

Reply:
IIRC during the initial showing, Mitch noted that the app will get posted webhooks for ALL builds under the azure org.  If you make the default ACTIVE we'll be check enforcing the entire world.

I believe one of the alternatives you offered was having a single common "these are what we care about" list in the `azure-sdk` repo.

Reply:
> I believe one of the alternatives you offered at the time was having a single common "these are what we care about" list in the `azure-sdk` repo.

I'm trying to make this solution general enough so that other teams in Microsoft can use it also (and potentially anyone using a mono-repo, including folks outside Microsoft).

Happy to make ```enabled: true``` the default, but obviously we want to support disabling as well. Having been enabled on a repo, having a kill switch without having to go through installing/uninstalling the app from the repo is desirable - especially in orgs like Microsoft where not everyone has repo admin rights.